src/main/java/org/alliancegenome/curation_api/jobs/executors/LoadFileExecutor.java

```diff
@@ -190,4 +189,10 @@ public void runCleanup(DiseaseAnnotationService service, BulkLoadFileHistory his
 		ph.finishProcess();
 		
 	}
+	
+	protected void failLoad(BulkLoadFile bulkLoadFile, Exception e) {
+		bulkLoadFile.setErrorMessage(e.getLocalizedMessage());
```

Comment:
I guess my point is that you will want to add these messages together. As in e.getMessage + e.getLocalizedMessage() + e.getCause().getMessage() doing null checks in case. This will give the best message otherwise someone is going to have to look in the logs anyway.

Reply:
There is also a "RootCasuse" somewhere that give an even better message... Something like this:
```
Throwable rootCause = e;
while (rootCause.getCause() != null && rootCause.getCause() != rootCause) {
     rootCause = rootCause.getCause();
}
rootCause.getMessage() + rootCause.getLocalizedMessage().;
```

Reply:
Ah, ok - done.  Used a Set to avoid duplication as in the test case e.getMessage() and e.getLocalizedMessage() return the same string.